deploy/role.yaml

```diff
@@ -77,3 +83,12 @@ rules:
   verbs:
   - list
   - watch
+- apiGroups:
+  - machineconfiguration.openshift.io
+  resources:
+  - machineconfigs  # The compliance remediation controller operates on MachineConfigs
+  verbs:
+  - list
+  - get
+  - patch
+  - create
```

Comment:
Let's add the "watch" verb here as well. This will be useful for the cache to be filled by Machineconfig objects.

Reply:
done